filecoin-hashers/src/types.rs

```diff
@@ -27,21 +28,49 @@ pub trait Domain:
     + Eq
     + Send
     + Sync
+    // TODO (halo): remove once we have Pasta GPU support.
```

Comment:
Note: `Domain: From<Fr> + Into<Fr>` is used as a stopgap to cause GPU code to panic when using Pasta fields; these bounds should be removed once the GPU work has finished.